examples/browser/browser.html

```diff
@@ -85,6 +85,7 @@ <h3>Results</h3>
                 span.log({ event : 'request_finished', response_body : json });
                 results.value = JSON.stringify(json);
             });
+            span.finish();
```

Comment:
I think this should be in the callback between lines 85 and 86, right?  We want the span to finish when all the inner async calls are done (I believe).